api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributesV2.java

```diff
@@ -0,0 +1,1006 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.trace.attributes;
+
+import static io.opentelemetry.common.AttributeKey.booleanKey;
+import static io.opentelemetry.common.AttributeKey.longKey;
+import static io.opentelemetry.common.AttributeKey.stringKey;
+
+import io.opentelemetry.common.AttributeKey;
+
+// DO NOT EDIT, this is an Auto-generated file from /templates/SemanticAttributesV2.java.j2
```

Comment:
I think it should be relatively straightforward to generate this in gradle using e.g. <https://docs.gradle.org/current/dsl/org.gradle.api.tasks.Exec.html>. Then we would not have to check in this file.

On the minus side, we would then require contributors to have a recent Python 3 installed (actually Docker currently which makes it un-buildable on Windows 🤔).

If we don't auto-generate the file, we should check during build that the file matches the generated output (does the tool support that for any generator currently or only markdown?)